frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -95,8 +94,6 @@ export const surveyLogic = kea<surveyLogicType>([
         actions: [
             surveysLogic,
             ['loadSurveys'],
-            pluginsLogic,
-            ['installPlugin'],
```

Comment:
Also ended up removing the plugin installation stuff since it doesn't really work so there's no point to it :D